doc/ir-semantics.text

```diff
@@ -2,8 +2,8 @@
 
 x        variables
 
-P ::= Segment  active   ↦ S, (Ds ↦ S)*   program:            active version and others
-S ::= start ↦ i,          I              segment:            instruction stream with dedicated start
+P ::= Segment  active   ↦ M, (Ds ↦ M)*   program:            active version and others
```

Comment:
btw. I use `T` for segmenT now, since it avoids confusion with Module and also I found Dm to look silly.